hoomd/md/methods.py

```diff
@@ -152,69 +166,96 @@ class NPT(_Method):
+    entries ``box_dof[3:]`` control the rescaling or the tilt factors xy, xz, 
+    and yz. All options also appropriately rescale particle coordinates and 
+    velocities.
 
-    The ``box_dof`` tuple controls the way the box is rescaled and updated. The first three elements ``box_dof[:2]``
-    controls whether the x, y, and z box lengths are rescaled and updated, respectively. The last three entries
-    control the rescaling or the tilt factors xy, xz, and yz. All options also appropriately rescale particle coordinates and velocities.
-
-    By default, the x, y, and z degrees of freedom are updated. [True,True,True,False,False,False]
+    By default, the x, y, and z degrees of freedom are updated. 
+    [True,True,True,False,False,False]
 
     Note:
-        If any of the diagonal x, y, z degrees of freedom is not being integrated, pressure tensor components
-        along that direction are not considered for the remaining degrees of freedom.
+        If any of the diagonal x, y, z degrees of freedom is not being 
+        integrated, pressure tensor components along that direction are not 
+        considered for the remaining degrees of freedom.
 
     For example:
 
-    - Specifying all couplings and x, y, and z degrees of freedom amounts to cubic symmetry (default)
-    - Specifying xy couplings and x, y, and z degrees of freedom amounts to tetragonal symmetry.
-    - Specifying no couplings and all degrees of freedom amounts to a fully deformable triclinic unit cell
-
-    :py:class:`NPT` Can also apply a constant stress to the simulation box. To do so, specify the symmetric
-    stress tensor *S* instead of an isotropic pressure *P*.
-
-    Note:
-        :py:class:`NPT` assumes that isotropic pressures are positive. Conventions for the stress tensor sometimes
-        assume negative values on the diagonal. You need to set these values negative manually in HOOMD.
-
-    :py:class:`NPT` is an integration method.
+    - Specifying all couplings and x, y, and z degrees of freedom amounts to 
+      cubic symmetry (default)
+    - Specifying xy couplings and x, y, and z degrees of freedom amounts to 
+      tetragonal symmetry.
+    - Specifying no couplings and all degrees of freedom amounts to a fully 
+      deformable triclinic unit cell
 
-    :py:class:`NPT` uses the proper number of degrees of freedom to compute the temperature and pressure of the system in
-    both 2 and 3 dimensional systems, as long as the number of dimensions is set before the :py:class:`NPT` command
-    is specified.
 
     For the MTK equations of motion, see:
 
     * `G. J. Martyna, D. J. Tobias, M. L. Klein  1994 <http://dx.doi.org/10.1063/1.467468>`_
     * `M. E. Tuckerman et. al. 2006 <http://dx.doi.org/10.1088/0305-4470/39/19/S18>`_
     * `T. Yu et. al. 2010 <http://dx.doi.org/10.1016/j.chemphys.2010.02.014>`_
-    * Glaser et. al (2013), to be published
+    *  Glaser et. al (2013), unpublished
```

Comment:
If it is unpublished and not to be published should we remove this?

Reply:
@jglaser How would you like us to document this going forward?